package-lock.json

```diff
@@ -7065,8 +7065,7 @@
         },
         "ansi-regex": {
           "version": "2.1.1",
-          "bundled": true,
-          "optional": true
+          "bundled": true
```

Comment:
Code changes contain no dependency updates/alterations. Do not recommend committing `package-lock.json`.

Reply:
Got it!